spec/bundler/fetcher/compact_index_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+describe Bundler::Fetcher::CompactIndex do
+  let(:downloader)  { double(:downloader) }
+  let(:remote)      { double(:remote, :cache_slug => "lsjdf") }
+  let(:display_uri) { URI("http://sampleuri.com") }
+  let(:compact_index) { described_class.new(downloader, remote, display_uri) }
+
+  # Testing private method. Do not commit.
```

Comment:
Good point. I missed removing that after I fixed the test to not be nearly so ugly.